libs/shared/src/consts/providers/channels/sms.ts

```diff
@@ -23,6 +23,14 @@ import { SmsProviderIdEnum } from '../provider.enum';
 import { ChannelTypeEnum } from '../../../types';
 
 export const smsProviders: IProviderConfig[] = [
+  {
+    id: SmsProviderIdEnum.Novu,
+    displayName: 'Novu SMS',
+    channel: ChannelTypeEnum.SMS,
+    credentials: [],
+    docReference: 'https://docs.novu.co/channels/sms/novu-sms',
```

Comment:
We son't have those docs. It will be done next cycle, right? Maybe for now use a link that we do have, in case its released before that ?